app/src/androidTest/java/org/fossasia/openevent/BookmarksTest.java

```diff
@@ -0,0 +1,156 @@
+package org.fossasia.openevent;
+
+import android.content.Context;
+import android.net.ConnectivityManager;
+import android.net.wifi.WifiManager;
+import android.support.test.InstrumentationRegistry;
+import android.support.test.espresso.contrib.RecyclerViewActions;
+import android.test.ActivityInstrumentationTestCase2;
+import android.util.Log;
+
+import org.fossasia.openevent.activities.MainActivity;
+import org.fossasia.openevent.data.Event;
+import org.fossasia.openevent.data.Microlocation;
+import org.fossasia.openevent.data.Session;
+import org.fossasia.openevent.data.Speaker;
+import org.fossasia.openevent.data.Sponsor;
+import org.fossasia.openevent.data.Track;
+import org.fossasia.openevent.data.Version;
+import org.fossasia.openevent.dbutils.DatabaseOperations;
+import org.fossasia.openevent.dbutils.DbContract;
+import org.fossasia.openevent.dbutils.DbHelper;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.action.ViewActions.click;
+import static android.support.test.espresso.action.ViewActions.swipeDown;
+import static android.support.test.espresso.action.ViewActions.swipeUp;
+import static android.support.test.espresso.assertion.ViewAssertions.matches;
+import static android.support.test.espresso.contrib.DrawerActions.openDrawer;
+import static android.support.test.espresso.matcher.ViewMatchers.isDisplayed;
+import static android.support.test.espresso.matcher.ViewMatchers.withContentDescription;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+import static android.support.test.espresso.matcher.ViewMatchers.withText;
+
+/**
+ * Created by MananWason on 7/30/2015.
+ */
+public class BookmarksTest extends ActivityInstrumentationTestCase2<MainActivity> {
+
+    private final Context context = InstrumentationRegistry.getInstrumentation().getTargetContext();
+
+    private DbHelper db;
+
+    public BookmarksTest() {
+        super(MainActivity.class);
+    }
+
+    @Override
+    protected void setUp() throws Exception {
+        super.setUp();
+        WifiManager wifiManager = (WifiManager) this.context.getSystemService(Context.WIFI_SERVICE);
```

Comment:
I'll have to implement Mockito but I wasn't sure how to do it. Could you help me with it?